frontend/src/locales/en.json

```diff
@@ -1,355 +1,358 @@
-    "Make Order":"Make Order",
-    "Info":"Info",
-    "View Book":"View Book",
+    "This is your trading avatar": "This is your trading avatar",
+    "Store your token safely": "Store your token safely",
+    "A Robot avatar was found, welcome back!": "A Robot avatar was found, welcome back!",
+    "Copied!": "Copied!",
+    "Generate a new token": "Generate a new token",
+    "Generate Robot": "Generate Robot",
+    "You must enter a new token first": "You must enter a new token first",
+    "Make Order": "Make Order",
+    "Info": "Info",
+    "View Book": "View Book",
 
 
     "MAKER PAGE - MakerPage.js": "This is the page where users can create new orders",
-    "Order":"Order",
-    "Customize":"Customize",
-    "Buy or Sell Bitcoin?":"Buy or Sell Bitcoin?",
-    "Buy":"Buy",
-    "Sell":"Sell",
-    "Amount":"Amount",
-    "Amount of fiat to exchange for bitcoin":"Amount of fiat to exchange for bitcoin",
-    "Invalid":"Invalid",
-    "Enter your preferred fiat payment methods. Fast methods are highly recommended.":"Enter your preferred fiat payment methods. Fast methods are highly recommended.",
-    "Must be shorter than 65 characters":"Must be shorter than 65 characters",
-    "Swap Destination(s)":"Swap Destination(s)",
-    "Fiat Payment Method(s)":"Fiat Payment Method(s)",
-    "You can add any method":"You can add any method",
-    "Add New":"Add New",
-    "Choose a Pricing Method":"Choose a Pricing Method",
-    "Relative":"Relative",
-    "Let the price move with the market":"Let the price move with the market",
-    "Premium over Market (%)":"Premium over Market (%)",
-    "Explicit":"Explicit",
-    "Set a fix amount of satoshis":"Set a fix amount of satoshis",
-    "Satoshis":"Satoshis",
-    "Let the taker chose an amount within the range":"Let the taker choose an amount within the range",
-    "Enable Amount Range":"Enable Amount Range",
+    "Order": "Order",
+    "Customize": "Customize",
+    "Buy or sell bitcoin?": "Buy or sell bitcoin?",
+    "Buy": "Buy",
+    "Sell": "Sell",
+    "Amount": "Amount",
+    "Amount of fiat to exchange for bitcoin": "Amount of fiat to exchange for bitcoin",
+    "Invalid": "Invalid",
+    "Enter your preferred fiat payment methods. Fast methods are highly recommended.": "Enter your preferred fiat payment methods. Fast methods are highly recommended.",
+    "Must be shorter than 65 characters": "Must be shorter than 65 characters",
+    "Swap Destination(s)": "Swap Destination(s)",
+    "Fiat Payment Method(s)": "Fiat Payment Method(s)",
+    "You can add any method": "You can add any method",
+    "Add New": "Add New",
+    "Choose a Pricing Method": "Choose a Pricing Method",
+    "Relative": "Relative",
+    "Let the price move with the market": "Let the price move with the market",
+    "Premium over Market (%)": "Premium over Market (%)",
+    "Explicit": "Explicit",
+    "Set a fix amount of Sats": "Set a fix amount of Sats",
```

Comment:
This is a tooltip displayed over the "explicit" radio control on the marker page. I would leave satoshis.